src/stylesheets/conversation.less

```diff
@@ -310,3 +310,27 @@
 
     }
 }
+
+.sk-accent-color-light {
```

Comment:
So `sk-accent-color-dark`, `sk-link-color-dark`, and `sk-branding-color-light` will just follow the default behavior?